recipe/meta.yaml

```diff
@@ -42,10 +42,9 @@ about:
   license: GPL-2
   summary: Calculates partial and semi-partial (part) correlations along with p-value.
   license_family: GPL2
-  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-2'  # [unix]
-  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'  # [win]
```

Comment:
I'm not sure why this is happening so often now. These files existed before and we are forced to ship a file in case of GPL. If this seems to be not an exception but the new standard we should talk to the bioc people and include a file in our pkgs on our own.

Reply:
This package was last updated in 2015, so there was never a license there. I wonder if R itself used to copy over license files if it knew what they should be.

Reply:
Mh, very wired.